<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageView2DPanelWithControls</class>
 <widget class="QDockWidget" name="ImageView2DPanelWithControls">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>334</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="features">
   <set>QDockWidget::AllDockWidgetFeatures</set>
  </property>
  <property name="windowTitle">
   <string>2D View</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="spacing">
            <number>1</number>
           </property>
           <item row="5" column="1">
            <widget class="QComboBox" name="comboBox_Interpolation">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Interpolation</string>
             </property>
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Nearest Neighbour</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Linear</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Cubic</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="5" column="2" colspan="3">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>1</number>
             </property>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Opacity</string>
               </property>
               <property name="maximum">
                <double>1.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_windowlevel_Syn">
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>link the window/level of the image planes</string>
               </property>
               <property name="text">
                <string>W/L</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_Visible">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Imge plane visibility</string>
               </property>
               <property name="text">
                <string>Vis</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_GradientPlay">
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Conponent Play</string>
               </property>
               <property name="text">
                <string>[]&gt;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_Play">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Slice Play</string>
               </property>
               <property name="text">
                <string>I&gt;</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="comboBox_Contents">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Image Source</string>
             </property>
             <item>
              <property name="text">
               <string>DWI</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QSlider" name="horizontalSlider_Gradient">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Gradient</string>
             </property>
             <property name="maximum">
              <number>0</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QSlider" name="horizontalSlider_SliceIndex">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Slice index</string>
             </property>
             <property name="maximum">
              <number>0</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::NoTicks</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="comboBox_Orientation">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Orientation</string>
             </property>
             <item>
              <property name="text">
               <string>Axial</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Sagittal</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Coronal</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLineEdit" name="lineEdit_Gradient">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Gradient Index</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QLineEdit" name="lineEdit_SliceIndex">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Slice Index</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QToolButton" name="toolButton_content_Syn">
             <property name="maximumSize">
              <size>
               <width>15</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Link the contents and conponent nuumbers of the image planes</string>
             </property>
             <property name="text">
              <string>S</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="autoRaise">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QToolButton" name="toolButton_orientation_Syn">
             <property name="maximumSize">
              <size>
               <width>15</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Link the orientation of the image planes</string>
             </property>
             <property name="text">
              <string>S</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QToolButton" name="toolButton_interpolation_Syn">
             <property name="maximumSize">
              <size>
               <width>15</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>link the interpolation of the image planes</string>
             </property>
             <property name="text">
              <string>S</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QVTKWidget" name="qvtkWidget" native="true">
      <property name="toolTip">
       <string>2D Image</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
